### ISSUE TITLE
Always roll 10

### ISSUE THREAD

**Issue by Stevenozzz** [cabarius/ToyBox#677]
**Describe the solution you'd like**
The same option as "always roll 1" or 20 but 10

**Additional context**
Additional dice roll option

**Comment by MysticBrightShadow** [comment]
This is already in the mod, on the "bag of tricks" tab, just scroll down.

**Comment by Stevenozzz** [comment]
There is, but out of combat only, isn't it

**Comment by MysticBrightShadow** [comment]
Yes, because this feature represents the table-top option of taking a 10, which typically meant doing an action more slowly, taking your time. Obviously in combat this isn't an option. I guess it wouldn't be too hard to add the cheat of always rolling 10 in combat. I could see it being useful to set your enemies to this actually.

**Comment by Stevenozzz** [comment]
Exactly, for combat there are only always roll 1 or 20. Wish we could just enter what kind of dice eitehr you or everybody rolls (for more deterministic  playthroughs)
